src/lang/english.txt

```diff
@@ -2720,7 +2720,7 @@ STR_RAIL_TOOLBAR_MAGLEV_CONSTRUCTION_CAPTION                    :Maglev Construc
 STR_RAIL_TOOLBAR_TOOLTIP_BUILD_RAILROAD_TRACK                   :{BLACK}Build railway track. Ctrl toggles build/remove for railway construction. Shift toggles building/showing cost estimate
 STR_RAIL_TOOLBAR_TOOLTIP_BUILD_AUTORAIL                         :{BLACK}Build railway track using the Autorail mode. Ctrl toggles build/remove for railway construction. Shift toggles building/showing cost estimate
 STR_RAIL_TOOLBAR_TOOLTIP_BUILD_TRAIN_DEPOT_FOR_BUILDING         :{BLACK}Build train depot (for buying and servicing trains). Shift toggles building/showing cost estimate
-STR_RAIL_TOOLBAR_TOOLTIP_CONVERT_RAIL_TO_WAYPOINT               :{BLACK}Convert rail to waypoint. Ctrl enables joining waypoints. Shift toggles building/showing cost estimate
+STR_RAIL_TOOLBAR_TOOLTIP_CONVERT_RAIL_TO_WAYPOINT               :{BLACK}Build railway waypoint. Ctrl enables joining waypoints. Shift toggles building/showing cost estimate
```

Comment:
It is a bit weird (but so are signals): you can't actually build them without first building a railway.

Now we can discuss if that actually make sense: why can't I just build a waypoint, but why can I build a station. It makes little sense, this inconsistency.

So the original text was more correct .. but in that sense, the signal text was wrong. So having it consistent makes sense. But it no longer suggests you first need a railway. But neither did signals.

It is a mess. Not your fault :)

I really don't care or mind either way btw; just an observation it is messy either way :)

Reply:
I don't think "build" infers that you can build it by itself. You can build a roof on a house, right?

"Add" might make sense, but I still prefer "Build."

Reply:
Ironically, you indeed say you build a roof ON a house. If you say: I am building a roof, it is a bit confusing .. you are building just a roof? Are you going to put it on the house later? So yeah, great example to highlight how weird this is :D

Reply:
Ask and ye shall receive. 🙂